ts/WoltLabSuite/Core/Component/Attachment/Entry.ts

```diff
@@ -59,8 +59,9 @@ function fileInitializationCompleted(element: HTMLElement, file: WoltlabCoreFile
     if (file.link !== undefined && file.filename !== undefined) {
       const link = document.createElement("a");
       link.href = file.link!;
-      link.classList.add("jsImageViewer");
+      link.dataset.fancybox = "attachments";
```

Comment:
Please use a different group so that these don’t mix with other images on the page.